lib/brainstem/api_docs/formatters/open_api_specification/version_2/field_definitions/presenter_field_formatter.rb

```diff
@@ -0,0 +1,147 @@
+require 'brainstem/api_docs'
+require 'brainstem/api_docs/formatters/abstract_formatter'
+require 'brainstem/api_docs/formatters/open_api_specification/helper'
+
+module Brainstem
+  module ApiDocs
+    module Formatters
+      module OpenApiSpecification
+        module Version2
+          module FieldDefinitions
+            class PresenterFieldFormatter < AbstractFormatter
```

Comment:
A little bit of this class shares procedural methods with `ResponseFieldFormatter`.

Should `ResponseFieldFormatter`, `EndpointParameterFormatter`, and `PresenterFieldFormatter` instead share these methods?